its/ruling/src/test/resources/expected/kubernetes/kubernetes-S1135.json

```diff
@@ -1,19 +1,4 @@
 {
-"kubernetes-project:ruling/src/test/resources/sources/kubernetes/helm/S5849/templates/deployment.yaml": [
-37
```

Comment:
I would expect that this and other issues will be reported. Together with SONARIAC-1148 it should work.
Or something is still missing?

Reply:
As I wrote above, some files lack values.yaml, some contain functions that we don't (fully) support yet